cocotb_test/simulator.py

```diff
@@ -1018,7 +1018,19 @@ def build_command(self):
 
 def run(**kwargs):
```

Comment:
@jonpovey Not better this way?:
```python
def run(simulator=None, **kwargs):
```

Reply:
Oh yeah, that's a little neater. Pushed amended version